crates/cheatcodes/Cargo.toml

```diff
@@ -14,6 +14,13 @@ exclude.workspace = true
 [lints]
 workspace = true
 
+[build-dependencies]
+vergen = { workspace = true, default-features = false, features = [
+    "build",
+    "git",
+    "gitcl",
+] }
```

Comment:
I assume this is fine, given that we also use this in other crates and the build overhead should be minimal